frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -322,21 +324,36 @@ export const MilestonesEditor = () => {
                         tasks={versionLists[version.id] || []}
                       />
                     </div>
-                    <div className={classes(css.milestoneFooter)}>
-                      <DeleteButton
-                        onClick={deleteVersionClicked(version.id)}
-                        href={modalLink(ModalTypes.DELETE_VERSION_MODAL, {
-                          id: version.id,
-                          roadmapId: roadmapId!,
-                        })}
-                      />
-                      <SettingsButton
-                        onClick={editVersionClicked(version.id, version.name)}
-                        href={modalLink(ModalTypes.EDIT_VERSION_MODAL, {
-                          id: version.id,
-                          name: version.name,
-                        })}
-                      />
+                    <div
+                      className={classes(css.milestoneFooter, {
+                        [css.loading]: disableDrag,
+                      })}
+                    >
+                      {disableDrag ? (
+                        <>
+                          <DeleteIcon /> <SettingsSharpIcon />
+                        </>
+                      ) : (
+                        <>
+                          <DeleteButton
+                            onClick={deleteVersionClicked(version.id)}
```

Comment:
```suggestion
                          <DeleteButton
                            disabled={disableDrag}
                            onClick={deleteVersionClicked(version.id)}
```

I'd rather add `disabled` property to the svg button, like normal buttons have. That way it can be used for all those buttons.

The disabled svg button would not use the link even if href was provided, and hover & pointer styles would not be applied.

To keep the height the same with or without a link, the `a` tag could be given `line-height: 1`